File: libs/utils/lib-core/src/main/java/com/yyxnb/what/core/interfaces/OnSelectListener.java
package com.yyxnb.what.core.interfaces;

import android.view.View;

/**
 * 选择回调
 *
 * @author yyx
 */
public interface OnSelectListener {

    /**
     * 点击回调
     *
     * @param v        view
     * @param position 当前位置
     * @param text     内容
     */
    void onClick(View v, int position, String text);
}

File: lib_common/src/main/java/com/yyxnb/common/interfaces/OnSelectListener.kt
package com.yyxnb.common.interfaces

import android.view.View

/**
 * 选择回调
 *
 * @author yyx
 */
interface OnSelectListener {
    /**
     * 点击回调
     *
     * @param v        view
     * @param position 当前位置
     * @param text     内容
     */
    fun onClick(v: View, position: Int, text: String?)
}